back/boxtribute_server/graph_ql/resolvers.py

```diff
@@ -398,13 +487,90 @@ def resolve_create_qr_code(*_, box_label_identifier=None):
     return create_qr_code(box_label_identifier=box_label_identifier)
 
 
+@mutation.field("changeDistributionEventState")
+@convert_kwargs_to_snake_case
+def resolve_change_distribution_event_state(*_, distribution_event_id, new_state):
+    authorize(permission="distro_event:write")
+    return change_distribution_event_state(distribution_event_id, new_state)
+
+
+@mutation.field("createDistributionEvent")
+@convert_kwargs_to_snake_case
+def resolve_create_distribution_event(*_, creation_input):
+    # TODO: Add permissions
+    authorize(permission="distro_event:write")
+    return create_distribution_event(user_id=g.user.id, **creation_input)
+
+
+@mutation.field("createDistributionSpot")
+@convert_kwargs_to_snake_case
+def resolve_create_distribution_spot(*_, creation_input=None):
+    authorize(permission="location:write")
+    return create_distribution_spot(
+        user_id=g.user.id, distribution_spot_input=creation_input
+    )
+
+
 @mutation.field("createBox")
 @convert_kwargs_to_snake_case
 def resolve_create_box(*_, creation_input):
     authorize(permission="stock:write")
     return create_box(user_id=g.user.id, **creation_input)
 
 
+@mutation.field("moveBoxToDistributionEvent")
+@convert_kwargs_to_snake_case
+def resolve_move_box_to_distribution_event(
+    mutation_obj, _, box_label_identifier, distribution_event_id
+):
+    authorize(permission="stock:write")
+    return move_box_to_distribution_event(box_label_identifier, distribution_event_id)
+
+
+@mutation.field("moveItemsFromBoxToDistributionEvent")
+@convert_kwargs_to_snake_case
+def resolve_move_items_from_box_to_distribution_event(
+    mutation_obj, _, box_label_identifier, distribution_event_id, number_of_items
+):
+    authorize(permission="unboxed_items_collection:write")
+    return move_items_from_box_to_distribution_event(
+        user_id=g.user.id,
+        box_label_identifier=box_label_identifier,
+        distribution_event_id=distribution_event_id,
+        number_of_items=number_of_items,
+    )
+
+
+@mutation.field("removePackingListEntryFromDistributionEvent")
+@convert_kwargs_to_snake_case
+def resolve_remove_packing_list_entry_from_distribution_event(
+    *_, packing_list_entry_id
+):
+
+    packing_list_entry = PackingListEntry.get(packing_list_entry_id)
```

Comment:
If the ID did not exist, peewee would already throw an exception here, hence the error handling in the next line is redundant. The peewee exception is automatically handled due to `format_database_errors` in `exceptions.py`, raising a bad-user-input GraphQL error.